.github/workflows/node-ci.js.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: MagicMirror Automated Testing
+
+on:
+  push:
+    branches: [ master, develop, ci-test ]
```

Comment:
the ci-test can only after the merge be removed?

Reply:
Yeah, I remove them as soon as everything is done.